src/server/JasmineGraphServer.cpp

```diff
@@ -2035,7 +2057,8 @@ void JasmineGraphServer::initiateOrgCommunication(std::string graphID, std::stri
         threadID++;
 
     }
-   
+
+    server_logger.log("Python servers initiated", "info");   
```

Comment:
There is only one python server we create per JasmineGraph cluster right?

Reply:
Removed in 58678b1a64d640daa8361e0ff3620faf4205adb9

Reply:
Done